tables/output_tables.Rmd

```diff
@@ -107,12 +109,25 @@ histology_df %>%
 
 ## Table 2: Patients with hypermutant or ultra-hypermutant tumors
 ```{r table 2}
+# read in clinvar variant annotations / add P/LP annotation
+clinvar <- read_tsv(clinvar_file) %>%
+  mutate(`Pathogenic germline variant` = case_when(ClinicalSignificance == "Pathogenic" ~ paste0(Name, " (P)"),
+                          ClinicalSignificance == "Likely pathogenic" ~ paste0(Name, " (LP)"))) %>%
+  select(Kids_First_Participant_ID, `Pathogenic germline variant`)
+
 # Read in coding TMB file, merge with histologies data
 tmb_hist <- read_tsv(file.path(data_dir, "pbta-snv-consensus-mutation-tmb-coding.tsv")) %>%
     # rename TSB to Kids_First_Biospecimen_ID
   rename(Kids_First_Biospecimen_ID = Tumor_Sample_Barcode) %>%
+  mutate(tmb = round(tmb,1)) %>%
   select(Kids_First_Biospecimen_ID, tmb) %>%
-  left_join(histology_df) 
+  left_join(histology_df) %>%
+  # add germline variants
+  left_join(clinvar) %>%     
+  # add "None detected" to pathogenic germline variant column
+  mutate(`Pathogenic germline variant` = case_when(is.na(`Pathogenic germline variant`) ~ "None detected",
+                                              TRUE ~ as.character(`Pathogenic germline variant`))
+  )
```

Comment:
```suggestion
  Spacing, and removing what should be an unnecessary `as.character`
```suggestion
  mutate(`Pathogenic germline variant` = case_when(is.na(`Pathogenic germline variant`) ~ "None detected",
                                                   TRUE ~ `Pathogenic germline variant`))
```
```